### PR TITLE
CI: fix metadrive test

### PR DESCRIPTION
**Pull request by bongbui321**
Changes to GHA yaml file due to change to pytest.

`-W ignore::DeprecationWarning` for this because: 
```
tools/sim/tests/test_metadrive_bridge.py:4: in <module>
    from openpilot.tools.sim.bridge.metadrive.metadrive_bridge import MetaDriveBridge
openpilot/tools/sim/bridge/metadrive/metadrive_bridge.py:3: in <module>
    from metadrive.component.sensors.base_camera import _cuda_enable
../.cache/pypoetry/virtualenvs/openpilot-oifF75oP-py3.11/lib/python3.11/site-packages/metadrive/__init__.py:4: in <module>
    import pygame  # it is important to import pygame after that
../.cache/pypoetry/virtualenvs/openpilot-oifF75oP-py3.11/lib/python3.11/site-packages/pygame/__init__.py:291: in <module>
    import pygame.pkgdata
../.cache/pypoetry/virtualenvs/openpilot-oifF75oP-py3.11/lib/python3.11/site-packages/pygame/pkgdata.py:25: in <module>
    from pkg_resources import resource_stream, resource_exists
../.cache/pypoetry/virtualenvs/openpilot-oifF75oP-py3.11/lib/python3.11/site-packages/pkg_resources/__init__.py:102: in <module>
    warnings.warn(
E   DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html

```

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Where are you seeing that error? I don't see it on the latest run on master.

**Comment by cl0cks4fe** [comment]
that test isn't running on master

**Comment by bongbui321** [comment]
I just tried `pytest tools/sim/tests/test_metadrive_bridge.py'

File: tools/sim/tests/test_metadrive_bridge.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
 import pytest
+import warnings
```
Comment (adeebshihadeh) [inline tools/sim/tests/test_metadrive_bridge.py:3]: make sure to leave a comment explaining what this is for

**Pull request merged**